pubtools/_quay/push_docker.py

```diff
@@ -728,15 +728,15 @@ def run(self) -> None:
                     LOG.error("There are failed push items. Cannot continue, running rollback.")
                 else:
                     LOG.error("Push of all index images failed, running rollback.")
-                self.rollback(backup_tags, rollback_tags)
+                self.rollback(amd64_backup_tags, rollback_tags)
```

Comment:
Manifest list can't be restored from amd64 tags.

Reply:
I updated amd64_backup_tags to have also manifest lists

Reply:
Uploading schem1 and schema2 manifests failed (errors are in testing doc). Do they need to be uploaded? I think only manifest list was uploaded before.

Reply:
I guess it's not absolutely necessary if there's manifest list, manifests shouldn't be garbage collected so soon. But if there isn't manifest list, then those should be uploaded. I'll try to investigate why it fails